### ISSUE TITLE
Ignoring corrupted files errors during ingest

### ISSUE THREAD

**Issue by Hgherzog** [allenai/rslearn_projects#79]
Some GCS files for sentinel 2 such as gs://gcp-public-data-sentinel-2/tiles/18/L/VQ/S2A_MSIL1C_20220823T151721_N0400_R125_T18LVQ_20220823T203046.SAFE/GRANULE/L1C_T18LVQ_A037445_20220823T152332/IMG_DATA/T18LVQ_20220824T151721_TCI.jp2 are actually an xml metadata file not a jp2  and thus can't be ingested. We want to check for this issue during the prepare phase. Until that is possible we will simply ignore errors during ingest and just continue as if it didn't happen.

**Comment by Hgherzog** [comment]
@favyen2 pretty sure youa dressed this can you confirm

**Comment by favyen2** [comment]
No, I fixed a different issue. I have another fix for images that say OPER instead of MSIL1C and should not be used that I still need to create PR for. Did this issue come up again?
